src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue

```diff
@@ -114,38 +114,54 @@ const isCheckbox = computed(() => (Array.isArray(props.modelValue)));
 const component = computed(() => (isCheckbox.value ? UiCheckbox : UiRadio));
 const componentName = computed<ComponentName>(() => (isCheckbox.value ? 'ui-checkbox' : 'ui-radio'));
 const errorClass = computed(() => (props.invalid
-  ? `${componentName.value}--has-error ui-list-item--has-error`
-  : ''));
+  ? [
+    `${componentName.value}--has-error`,
+    'ui-list-item--has-error',
+  ]
+  : []));
+
 function unfocusSuffix(event: KeyboardEvent) {
   if (event.key !== 'ArrowLeft') return;
-  const suffixBtn = event.target as HTMLElement;
-  const input: HTMLInputElement | null | undefined = suffixBtn.closest('.ui-multiple-answer-item')?.querySelector('input');
-  input?.focus();
+  const suffixButton = event.target as HTMLElement;
+  const input: HTMLInputElement | null | undefined = suffixButton.closest('.ui-multiple-answer-item')?.querySelector('input');
+  if (input) {
+    event.preventDefault();
+    focusElement(input);
+  }
 }
 function focusSuffix(event: KeyboardEvent) {
   if (event.key !== 'ArrowRight') return;
   const input = event.target as HTMLInputElement;
-  const suffixBtn: HTMLElement | null | undefined = input.closest('.ui-multiple-answer-item')?.querySelector('.ui-list-item-suffix');
```

Comment:
`.ui-list-item-suffix` 🛑  you should add class to this element :) and use it in this function. When you use `.ui-list-item-suffix` you create dependency and when you change something in `.ui-list-item-suffix` you should change it here :(